src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/organizationFiltersAndSampling.tsx

```diff
@@ -1,27 +1,57 @@
 import React from 'react';
 
 import {t} from 'app/locale';
+import {Organization} from 'app/types';
+import {DynamicSamplingRules, DynamicSamplingRuleType} from 'app/types/dynamicSampling';
 import AsyncView from 'app/views/asyncView';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import TextBlock from 'app/views/settings/components/text/textBlock';
 import PermissionAlert from 'app/views/settings/organization/permissionAlert';
 
 import TransactionRules from './transactionRules';
 
-type Props = AsyncView['props'];
+type Props = AsyncView['props'] & {
+  organization: Organization;
+};
 
-type State = AsyncView['state'];
+type State = AsyncView['state'] & {
+  transactionRules: DynamicSamplingRules;
+};
 
 class OrganizationFiltersAndSampling extends AsyncView<Props, State> {
   getTitle() {
     return t('Filters & Sampling');
   }
 
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      transactionRules: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getTransactionsRules();
+  }
+
+  getTransactionsRules() {
```

Comment:
Should this use `getDerivedStateFromProps()`? Or should you also have a componentDidUpdate() to ensure the state is synced when the organization changes?